data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"Is this it? Is the game over now that I completed the free worlds campaign?"`
```

Comment:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all parts of the game have been implemented yet."``
```

Also, Free Worlds should be capitalized if this isn't reverted. ;)

Reply:
A new player may interpret "the rest of the galaxy" to mean "everything beyond human space" and "not all parts of the game have been implemented yet" to mean "everything beyond human space is not implemented yet"

Reply:
Something like;
'I want to explore beyond human space, but apparently it hasn't all been implemented in the game yet,' perhaps?